src/lib.rs

```diff
@@ -268,5 +268,11 @@ pub use self::message::*;
 pub mod constants;
 pub use self::constants::*;
 
-pub(crate) use self::utils::traits::*;
-pub(crate) use netlink_packet_utils as utils;
+pub mod nla;
+pub use self::nla::*;
+
+pub mod parsers;
+pub use self::parsers::*;
```

Comment:
Since we are breaking stuff, let's not expose the module patch to public allowing us to re-organize the code in the future. Try expose everything as top level data type or function and don't use `::*`.

```
pub use self::nla::{NlaBuffer, DefaultNla, NlaIterator};
pub use self::parsers::{parse_u8, ...};
```